docs/modules/development/pages/rest/health_rest.adoc

```diff
@@ -0,0 +1,92 @@
+
+= Health Rest Service
+
+Service that allows health rest calls on all _HealthCheck_ components available .
```

Comment:
```suggestion
Service that allows health rest calls on all available health check components.
```

Reply:
Sorry, I didn't notice this before, but I checked the main docs, and we tend to use "health check".